File: src/test/java/ev/projects/GameMother.java
package ev.projects;

import java.util.List;

public class GameMother {

    Game new2PlayerGame(Center center) {
        return new2PlayerGame(new PlayerMother().newPlayer(), center);
    }

    Game new2PlayerGame(Player player1) {
        return new2PlayerGame(player1, new Center());
    }

    Game new2PlayerGame(Player player1, Center center) {
        return new Game(List.of(player1, new PlayerMother().newPlayer()), center);
    }
}

File: python/lib/test/game_mother.py
from azul.game import Game
from azul.center import Center
from player_mother import PlayerMother

class GameMother:
    @staticmethod
    def new_2_player_game(center=None, player1=None):
        if center is None:
            center = Center()
        if player1 is None:
            player1 = PlayerMother().new_player()
        player2 = PlayerMother().new_player()
        return Game([player1, player2], center)